### PR TITLE
feat: return informative help messages and urls for KYC

### PR DESCRIPTION
**Pull request by jowparks**
## Summary
1. Returns better/more discrete error messages for users when calculating eligibility.
2. Returns associated `help_url` if merited
## Testing Plan
tests pass/test in staging
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: src/redemptions/redemption.service.ts [hunk 1]
```diff
@@ -360,13 +367,35 @@ export class RedemptionService {
...
+          .map((c) => this.hasBannedCountry(c.id_issuing_country))
+          .some((c) => c);
+      }
+      if (kycCountries && hasBannedCountry) {
+        return {
+          eligible: false,
+          reason: `A country associated with your KYC attempt is banned: ${kycCountries
+            .map((c) => c.id_issuing_country)
+            .join(', ')}`,
+          helpUrl: '',
```
Comment (jowparks) [inline src/redemptions/redemption.service.ts:398]: Do we want to add a help url here? There is no action we can take

File: src/redemptions/redemption.service.ts [hunk 2]
```diff
@@ -390,17 +420,60 @@ export class RedemptionService {
...
+        transactions.map((t) => t.user_id),
+      );
+      const graffitis = users.map((u) => u.graffiti);
+      return {
+        eligible: false,
+        reason: `You cannot KYC for more than one account. 
+        You have completed KYC with other graffitis${
+          graffitis ? ': ' + graffitis.join(', ') : ''
+        }`,
+        helpUrl: '',
```
Comment (jowparks) [inline src/redemptions/redemption.service.ts:472]: help url? We aren't going to unban right?

File: src/redemptions/redemption.service.ts [hunk 3]
```diff
@@ -360,13 +367,35 @@ export class RedemptionService {
         return {
           eligible: false,
```
Comment (jowparks) [inline src/redemptions/redemption.service.ts:368]: @skylarrichard123

File: src/redemptions/redemption.service.ts [hunk 4]
```diff
@@ -390,17 +420,60 @@ export class RedemptionService {
...
+      const transactions = await this.jumioTransactionService.findByWorkflowIds(
+        repeatedFaceWorkflowIds,
+      );
+      const users = await this.usersService.findManyById(
+        transactions.map((t) => t.user_id),
+      );
+      const graffitis = users.map((u) => u.graffiti);
+      return {
+        eligible: false,
+        reason: `You cannot KYC for more than one account.
```
Comment (NullSoldier) [inline src/redemptions/redemption.service.ts:468]: This is bad you can dox other players by uploading a picture of someone elses face. It should not mention the other graffitis.

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**